js/mail.js

```diff
@@ -265,7 +265,10 @@ var Mail = {
 					if (currentAddress && email === currentAddress.emailAddress) {
 						label = t('mail', 'you');
 					}
-					return memo + '<span title="' + email + '">' + label + '</span>';
+					var title = t('mail', 'Send message to {email}', {email: email});
+					memo += '<span class="tipsy-mailto" title="' + title + '"><a class="link-mailto" data-email="' + email + '">' + label + '</a></span>';
```

Comment:
I guess I tried that and the styling was broken - only spans are properly aligned afaik ...